.github/workflows/release_pypi.yml

```diff
@@ -184,14 +99,13 @@ jobs:
       - name: Create GitHub Release
         env:
           GITHUB_TOKEN: ${{ github.token }}
-        # Create a GitHub Release *draft*. The description can be changed later, as for now
+        # Create a tag and a GitHub Release. The description can be changed later, as for now
         # we don't define it through this workflow.
         run: >-
           gh release create
           '${{ github.ref_name }}'
```

Comment:
I noticed that this can't work - `ref_name` will be `master` instead of a tag name. I'll have to think of something